capa/features/common.py

```diff
@@ -272,6 +272,17 @@ def __str__(self):
         return f'substring("{self.value}", matches = {matches})'
 
 
+class Registry(String):
+    pass
```

Comment:
let's discuss the requirements and sketch a design for how this feature should look and act in #1558 

this should include how the rule syntax changes, the format of the feature and properties of the feature, and at least two example rules showing how the feature would be used. 

once we're happy with the design, then please open a standalone PR with the Registry feature.